.swiftlint.yml

```diff
@@ -0,0 +1,8 @@
+disabled_rules:
+  - line_length
+  - identifier_name
+  - type_body_length
+  - cyclomatic_complexity
```

Comment:
Would be nice to enable the `cyclomatic_complexity` also later, but needs more work